test/app/nginx.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: web
 spec:
   serviceName: "nginx"
-  replicas: 1
+  replicas: 6
```

Comment:
Is this change expected or included in PR accidentally?

Reply:
Accidentally. Thanks, reverting...